internal/db/seeds/integration.sql

```diff
@@ -17,7 +17,9 @@ $do$;
 
 -- Seed the data
 INSERT INTO accounts(id, account_number, org_id)
-VALUES (1, '1', '1')
+VALUES
+  (1, '1', '1'),
+  (2, '2', '2')
```

Comment:
I tried that, but it causes failure in account test, which is bit unfortunate, but I did not want to go to the rabbit hole and fix it, so this is what I come up with, feel free to use, but ofc I'm happy to just drop it ^_^ #598

Reply:
eh your account tests do not fail :thinking: so I just did something random, I shall stop try to code it appears :facepalm: